src/models/user.rs

```diff
@@ -179,19 +185,21 @@ impl User {
     /// Converts this `User` model into an `EncodablePrivateUser` for JSON serialization.
     pub fn encodable_private(
         self,
+        email: Option<String>,
+
         email_verified: bool,
         email_verification_sent: bool,
-    ) -> EncodablePrivateUser {
+    ) -> CargoResult<EncodablePrivateUser> {
```

Comment:
Can you explain why this function needed to be changed to return a `Result`? It doesn't look like it's necessary to me because I don't see anywhere returning an `Err` variant. Am I missing something?

Reply:
It isn't necessary. I was previously reading this `email` variable from the database with an `Err`

https://github.com/rust-lang/crates.io/blob/3544b91086e822607cfeadda57b261e66112ba52/src/models/user.rs#L86

variant result before realizing [that was already "done"](https://github.com/rust-lang/crates.io/blob/3544b91086e822607cfeadda57b261e66112ba52/src/controllers/user/me.rs#L28-L37) and the email could simply be passed to the `src/models/users/encodable_private` function as an argument.
I definitely forgot to undo that modification to the return type - will do so now